packages/mariadb-10.11-mroonga/debian/compat

```diff
@@ -0,0 +1 @@
+10
```

Comment:
We can use 13 for Debian GNU/Linux bookworm or later: https://packages.debian.org/search?keywords=debhelper

```suggestion
13
```